hawkular-wildfly-monitor/src/main/java/org/hawkular/agent/monitor/inventory/ResourceConfigurationPropertyType.java

```diff
@@ -16,16 +16,9 @@
  */
 package org.hawkular.agent.monitor.inventory;
 
-public abstract class ResourceConfigurationPropertyType<RT extends ResourceType<?, ?, ?, ?>> extends NamedObject {
+public abstract class ResourceConfigurationPropertyType extends NamedObject {
 
-    private final RT resourceType;
-
-    public ResourceConfigurationPropertyType(ID id, Name name, RT resourceType) {
```

Comment:
why remove this? Its possible that given a res config prop type, we'll want to know what resource type it was associated with. Without this link we lose that context.

same with Operation